src/Bot.ts

```diff
@@ -152,4 +155,16 @@ export class Bot extends builder.UniversalBot {
             callback(null, null, 200);
         };
     }
+
+    // set incoming event to any because membersAdded is not a field in builder.IEvent
+    private getMessageReactionsHandler(bot: builder.UniversalBot): (event: any) => void {
+    return async function(event: any): Promise<void> {
+            let session = await loadSessionAsync(bot, event);
+            if (event.reactionsAdded && event.reactionsAdded[0].type && event.reactionsAdded[0].type === "like") {
+                session.send(Strings.like_message);
+            } else {
+                session.send(Strings.remove_like_message);
```

Comment:
This should not be else. Check reactionsRemoved explicitly (like you did for reactionsAdded), so that this works when we have a new kind of reaction.

Reply:
Done.